README.md

```diff
@@ -38,6 +38,18 @@ When `prebuild-install` is run via an `npm` script, options
 `--build-from-source`, `--debug` and `--download`, may be passed through via
 arguments given to the `npm` command.
 
+### Custom binaries
+The end user can override binary download location through environment variables in their .npmrc file.  
+The variable needs to meet the mask `% your package name %_binary_site`. For example:
```

Comment:
env name wrong here and below

Reply:
oh shoot, i knew i forgot something! thanks for noticing